fieldsolver/derivatives.cpp

```diff
@@ -127,6 +154,17 @@ void calculateDerivatives(
       }
    }
 
+   // Ex for electron charge separation  
+   if ((sysBoundaryFlag == sysboundarytype::NOT_SYSBOUNDARY) || (sysBoundaryLayer == 1)) {
```

Comment:
Not needed for sysBoundaryLayer==1

Reply:
That's a leftover from the thought to have responsive boundary "baths" of electrons. This shouldn't affect Vlasov acceleration steps, right?

Reply:
No as it won't be used, just pointless calculation on the boundary cells. Easy enough to put back in if needed in the future.